src/Html.php

```diff
@@ -364,12 +369,16 @@ public static function endTag($name): string
      * corresponding attribute will not be rendered. See {@see renderTagAttributes()} for details on how attributes
      * are being rendered.
      *
+     * @psalm-param HtmlOptions|array<empty, empty> $options
+     *
      * @throws JsonException
      *
      * @return string The generated style tag.
      */
     public static function style(string $content, array $options = []): string
     {
+        /** @psalm-var HtmlOptions $options */
+        $options['encode'] ??= false;
```

Comment:
It could be that there's a special encode needed for raw text elements: https://html.spec.whatwg.org/multipage/syntax.html#cdata-rcdata-restrictions

Reply:
No, that's not true. Can't really be escaped.